### PR TITLE
Switch to literal string interpolation

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: stratis-storage/project#456

### CODE REVIEW

File: src/dbus_python_client_gen/_invokers.py [hunk 1]
```diff
@@ -307,10 +307,11 @@ def dbus_func(proxy_object, func_args):  # pragma: no cover
                 param_list = list(arg_names_set)
                 arg_list = list(func_args.keys())
                 err_msg = (
-                    "Argument keywords passed (%s) did not match "
-                    'argument keywords expected (%s) for method "%s" '
-                    'belonging to interface "%s"'
-                ) % (", ".join(arg_list), ", ".join(param_list), name, interface_name)
+                    f"Argument keywords passed ({', '.join(arg_list)}) did not match "
+                    f"argument keywords expected ({', '.join(param_list)})' "
+                    f"for method {name}' "
```
Comment (mulkieran) [inline src/dbus_python_client_gen/_invokers.py:312]: I think that this line should be ```f'for method "{name}" '``` and that there is an extra ```'``` in the line above.

**Review by mulkieran — CHANGES_REQUESTED** [review]
I think I've spotted one.

File: src/dbus_python_client_gen/_invokers.py (hunk 1 above)
Comment (bgurney-rh) [reply src/dbus_python_client_gen/_invokers.py]: OK; I see it.  I kept missing this before; for cases where a `"` character is intentionally being printed, I have to surround the string with `'` characters.

File: src/dbus_python_client_gen/_invokers.py [hunk 2]
```diff
@@ -307,10 +307,11 @@ def dbus_func(proxy_object, func_args):  # pragma: no cover
                 param_list = list(arg_names_set)
                 arg_list = list(func_args.keys())
                 err_msg = (
-                    "Argument keywords passed (%s) did not match "
-                    'argument keywords expected (%s) for method "%s" '
-                    'belonging to interface "%s"'
-                ) % (", ".join(arg_list), ", ".join(param_list), name, interface_name)
+                    f"Argument keywords passed ({', '.join(arg_list)}) did not match "
+                    f"argument keywords expected ({', '.join(param_list)})' "
```
Comment (mulkieran) [inline src/dbus_python_client_gen/_invokers.py:311]: One last change, there is still a stray ```'``` inside the string.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Just one more...

File: src/dbus_python_client_gen/_invokers.py (hunk 2 above)
Comment (bgurney-rh) [reply src/dbus_python_client_gen/_invokers.py]: Resolved.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**